app/src/main/java/ch/epfl/cs311/wanderwave/model/spotify/SpotifyController.kt

```diff
@@ -99,6 +101,113 @@ class SpotifyController(private val context: Context) {
     }
   }
 
+  /**
+   * get the element under the tab "listen recently"
+   *
+   * @return a Flow of ListItem
+   * @author Menzo Bouaissi
+   * @since 2.0
+   * @last update 2.0
+   */
+  @OptIn(FlowPreview::class)
+  fun getTrack(): Flow<ListItem> {
+    return callbackFlow {
+      val callResult =
+          appRemote?.let { it ->
+            it.contentApi
+                .getRecommendedContentItems(ContentApi.ContentType.DEFAULT)
+                .setResultCallback {
+                  for (i in it.items) if (i.uri == "spotify:section:0JQ5DAroEmF9ANbLaiJ7Wx")
+                      trySend(i)
+                  // TODO checkt if "listen recently playlist" the same for everyone
```

Comment:
I think we should discuss at the next meeting whether we should use the "recent songs" playlist or if we should let user choose one of their own playlist.

Reply:
Also might be good to have that URI as a string resource